### DISCUSSION TITLE
Side Panel keeps content open when closed

### DISCUSSION THREAD

**Discussion by blunt4286** [zen-browser/desktop#2378] [Ideas]
Apologies if this has been posted already, i did a search and found nothing.

It would be really cool if the side panel kept its content loaded.
Currently, I can open youtube music and start playing a song and if I close the side panel, the music carries on.
However, if i want to change a track, opening the side panel causes youtube music to reload.

Other than this, really enjoying the browser, a big thank you to the team 😄

**Comment by Eragonfr** [comment]
I guess that the web panel reset its content when you open it.
It might even be a bug.
